tests/behat/features/web/admin/server_maintenance.feature

```diff
@@ -14,6 +14,9 @@ Feature: Admin Server Maintenance
     When I GET "/admin/maintain/list"
     Then the client response should contain "Delete APKs"
     And the client response should contain "Delete extracted files"
+    And the client response should contain "Delete log files"
+    And the client response should contain "Archive logs files"
+
 
   Scenario: As a valid admin I want to be able to remove the APKs through the backend
```

Comment:
Would be nice to have tests for this section
![image](https://user-images.githubusercontent.com/40868718/89387063-42de6900-d702-11ea-8d61-8c107bcb668d.png)